### ISSUE TITLE
[Accessibility] Card with tapAction cannot be executed thru ENTER/SPACEBAR

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3196]
## Version

`master` branch.

## Describe the bug

On a card with `tapAction` set, after <kbd>TAB</kbd> into the card component (root container), pressing <kbd>ENTER</kbd> or <kbd>SPACEBAR</kbd> key should execute the card.

## Steps to reproduce

1. Talk to MockBot
1. Type `receiptcard`
1. Press <kbd>TAB</kbd> key to move the focus to the card container
1. Press <kbd>ENTER</kbd> or <kbd>SPACEBAR</kbd>

## Expected behavior

It should open a new window, as stated by the `receiptcard` command.

## Additional context

[Bug]
